src/DetailsView/components/test-view-container.tsx

```diff
@@ -72,6 +82,15 @@ export const TestViewContainer = NamedFC<TestViewContainerProps>('TestViewContai
         automatedChecksCardSelectionMessageCreator:
             props.deps.automatedChecksCardSelectionMessageCreator,
         needsReviewCardSelectionMessageCreator: props.deps.needsReviewCardSelectionMessageCreator,
+        assessmentCardSelectionMessageCreator:
```

Comment:
We can use the `assessment-functionality-switcher` instead. An example of what that would look like can be seen in `iframe-warning`.